yarn-project/p2p/src/client/p2p_client.ts

```diff
@@ -161,6 +169,9 @@ export class P2PClient implements P2P {
       this.log(`Next block ${blockToDownloadFrom} already beyond latest block at ${this.latestBlockNumberAtStart}`);
     }
 
+    // publish any txs in TxPool after its doing initial sync
+    this.syncPromise = this.syncPromise.then(() => this.publishStoredTxs());
```

Comment:
Do we need to this? Transactions are propagated as soon as they are received. Do they need to be re-published when we start up?

Reply:
The use case I was thinking of: we're the only P2P client on the network, we get a bunch of transaction but no one to tell about them. We shut down and come back some time later, so we broadcast any transactions stored locally. WDYT?

Reply:
I think it's fine for now. We can re-visit later.